src/identity/components/GlobalHeader/constants/index.ts

```diff
@@ -0,0 +1 @@
+export const GLOBAL_HEADER_HEIGHT = 60
```

Comment:
I think a similar comment added saying change `$globalheader-height` in `GlobalHeaderStyles.scss` would be useful

Reply:
👍 Good idea.